scripts/setup-project.sh

```diff
@@ -48,9 +48,9 @@ create_iotc_azrtos_symlinks() {
   source_dir="${1:-../../../iotc-azrtos-sdk/}"
   target_dir=${2:-iotc-azrtos-sdk/}
 
-  if [ ! -d $target_dir ]; then
-    mkdir -p $target_dir
-  fi
+  # Make sure tha the script is idempotent
```

Comment:
It's just missing a "t". Make sure tha**t** the script is idempotent (it can be re-run safely) by clearing what may have been done in the previous run. I'll fix the typo.